lib/rubocop/cop/style/endless_method.rb

```diff
@@ -15,36 +18,94 @@ module Style
       # * allow_single_line (default) - only single line endless method definitions are allowed.
       # * allow_always - all endless method definitions are allowed.
```

Comment:
By the way, is there any difference between `EnforcedStyle: allow_always` and `Enabled: false`?

Reply:
Ah, indeed it is!
It's possible that the option will be deprecated, but the incompatibility is annoying.